src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -121,6 +126,10 @@ run_teuthology_tests() {
         export RGW=1
     fi
 
+    export COVERAGE_ENABLED=true
+    export COVERAGE_FILE=.coverage.mgr.dashboard
+    find . -iwholename "*.coverage.mgr.dashboard*" -exec rm -rf {} \;
```

Comment:
given you just defined that var, and if you're looking just for files (`-delete` and optionally `-type f`):
```suggestion
    find . -iname "*${COVERAGE_FILE}*" -delete
```

Reply:
Done.